internal/opa/output.go

```diff
@@ -47,12 +47,14 @@ type TemplateFields = struct {
 	Title       string
 	Description string
 	ShortName   string
+	DocsUrl     string
 }
 
 var templates = map[string]string{
 	"text": hd.Doc(`
 		{{ .TrimmedPath }}.{{ .ShortName }} ({{ .WarnDeny }})
-		{{ .Title }}
+		{{ if .DocsUrl }}{{ .DocsUrl }}
+		{{ end }}{{ .Title }}
```

Comment:
Not important at all, just wanted to mention this is another way of doing this:
```suggestion
		{{ with .DocsUrl }}{{ . }}
		{{ end }}{{ .Title }}
```

Reply:
I like it.